### ISSUE TITLE
[FEATURE] GM-Facing "Copy Scan Results" button

### ISSUE THREAD

**Issue by u-lulu** [massif-press/compcon#2179]
**Is your feature request related to a problem? Please describe.**
Whenever our team of players chooses to Scan an enemy (which is rare), it takes a bit for the GM to properly transmit all of that information to us. With the release of the GMS CHOMOLUNGMA in Operation Solstice Rain, scanning will likely become much more common in our future sessions.

**Describe the solution you'd like**
It would be nice for there to be a GM-Facing 'Scan' button in the mission runner that copies a text statblock of a unit to the clipboard, including any information that would be obtained from scanning. So whenever a unit is scanned, the GM can simply press that button and paste the information in chat.

**Describe alternatives you've considered**
Screenshots are the next best thing, but this can be clunky, as particularly complex NPCs take multiple screenshots just to give all relevant information.

**Comment by TheQuazman** [comment]
I made a PR for a "start" of this.  I is just a basic plantext statblock and not the full feature information that a scan would give, but it's a start.  

As a not for future improvements/implementation, might be a cool to make a dedicated "Scan" dialog and be able to select/deselect what info you want to share so it could be more general for things like a scan, lotus projector, athena, etc.  But that would be much more effort than the current PR I made utilizing the existing NPC stat block generation.
